index.js

```diff
@@ -17,70 +19,111 @@ const productB = {
+            achievementsBadge: '',
+            achievementsBadgeTwo: false,
+            totalRevenue: 0,
+            totalCommission: 0,
+        }
+    }
+}
 
-// Showing the revenue and commission in the HTML
-const revenueBar = document.getElementById('revenue-bar')
-const commissionBar = document.getElementById('commission-bar')
+// Store sales data in local storage
+function updateSalesData() {
+    localStorage.setItem('salesData', JSON.stringify(salesData));
+}
 
-renderSales()
+// Reset saved sales data values
+function resetSalesData() {
+    salesData.totalSales = '';
+    salesData.timesClicked = 0;
+    salesData.achievementsUnlocked = 0;
+    salesData.achievementsBadge = '';
+    salesData.achievementsBadgeTwo = false;
+    salesData.totalRevenue = 0;
+    salesData.totalCommission = 0;
+    updateSalesData(); // update the saved sales data in local storage
+    renderSales(); // render updated data
+}
 
-// Sales button actions
+
+// *** Moved 6 const variables for DOM updates into renderSales func - Matt
+
+// *** Sales Calculations
 function fixSale(salesProduct) {
-    totalSales += salesProduct.emoji
-    totalRevenue += salesProduct.revenue
-    totalCommission += salesProduct.commission
-    timesClicked += 1
+    // changed to update these variables in salesData local storage object instead of previous global variables - Matt
+    salesData.totalSales += salesProduct.emoji
+    salesData.totalRevenue += salesProduct.revenue
+    salesData.totalCommission += salesProduct.commission
+    salesData.timesClicked += 1
     checkAchievements()
+    updateSalesData() // update sales data in local storage - Matt
     renderSales()
 }
 
-// Show sales
+// Calculate achievements
+function checkAchievements() {
+    if (salesData.timesClicked === 1) {
+        salesData.achievementsUnlocked += 1
+        salesData.achievementsBadge += '🔔'
+    } else if (salesData.totalRevenue >= 2500 && salesData.achievementsBadgeTwo === false) {
+        salesData.achievementsUnlocked += 1
+        salesData.achievementsBadge += '💰'
+        salesData.achievementsBadgeTwo = true
+    } else if (salesData.timesClicked === 15) {
+        salesData.achievementsUnlocked += 1
+        salesData.achievementsBadge += '🏆'
+    }
+}
+
+// Render salesData 
 function renderSales() {
-    salesHeader.textContent = `Live Sales - ${timesClicked}`
-    salesBar.textContent = totalSales
-    achievementsHeader.textContent = `Live Achievements - ${achievementsUnlocked}`
-    achievementsBar.textContent = achievementsBadge
-    revenueBar.textContent = `$ ${totalRevenue}`
-    commissionBar.textContent = `$ ${totalCommission}`
+    // *** Moved these six variable here from global scope since we only need to access them from within this function
+    // *** Alternatively, we could remove the consts altogether and apply salesData to the individual document selectors
+    // *** for example: document.getElementById('sales-header').textContent = timesClicked 
```

Comment:
I think removing the consts would be better as well instead of having a bunch of variables that only get used to do one thing.